clientv3/integration/kv_test.go

```diff
@@ -381,6 +381,36 @@ func TestKVRange(t *testing.T) {
 				{Key: []byte("fop"), Value: nil, CreateRevision: 9, ModRevision: 9, Version: 1},
 			},
 		},
+		// fetch entire keyspace using WithFromKey
+		{
+			"\x00", "",
```

Comment:
It'll fail because no key is defined. No one complains about Get(ctx, "", WithFromKey())